Guesser/PropelORMFieldGuesser.php

```diff
@@ -279,26 +280,32 @@ protected function getOptions($type, $dbType, $model, $fieldPath, $filter = fals
         $columnName = $resolved['field'];
 
         if ((\PropelColumnTypes::BOOLEAN == $dbType || \PropelColumnTypes::BOOLEAN_EMU == $dbType) &&
-            (preg_match("#^choice#i", $type) || preg_match("#choice$#i", $type))) {
-            return array(
+            preg_match("#ChoiceType$#i", $type)) {
+            $options = array(
                 'choices' => array(
-                   0 => 'boolean.no',
-                   1 => 'boolean.yes'
+                    'boolean.no' => 0,
+                    'boolean.yes' => 1
```

Comment:
@sescandell current choice type implementation has inverted key - value in  `choices` option?